2.4/examples/replica/mongodb-clustered.json

```diff
@@ -43,12 +43,12 @@
     },
     {
       "name": "MONGODB_SERVICE_NAME",
-      "description": "The name of the MongoDB Service (used to DNS lookup, default: 'mongodb')",
```

Comment:
we want to show this in the console (iow. what we will create when user don't change it).